src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.html

```diff
@@ -2,54 +2,97 @@
   <div class="manage-materials">
     <div class="tabs">
       <ul id="materials-title">
-        <li [ngClass]="{'active': slide === 1}" (click)="changeSlide(1)">Materials</li>
-        <li [ngClass]="{'active': slide === 2}" (click)="changeSlide(2)">URLs</li>
-        <li [ngClass]="{'active': slide === 3}" (click)="changeSlide(3)">Notes</li>
+        <li [ngClass]="{ active: slide === 1 }" (click)="changeSlide(1)">
+          Materials
+        </li>
+        <li [ngClass]="{ active: slide === 2 }" (click)="changeSlide(2)">
+          URLs
+        </li>
+        <li [ngClass]="{ active: slide === 3 }" (click)="changeSlide(3)">
+          Notes
+        </li>
       </ul>
     </div>
     <div class="materials-slides">
-      <div [@fadeIn] *ngIf="slide === 1" class="slide slide1">
+      <div
+        [@fadeIn]
+        *ngIf="slide === 1"
+        class="slide slide1"
+        (dragenter)="dragenter($event)"
+        (dragover)="dragover($event)"
+        (drop)="handleDrop($event)"
+      >
         <div *ngIf="solutionUpload" class="solutions-warning">
-          Note: While the upload of solution files are not prohibited, they won't be included during a learning
-          object's download.
+          Note: While the upload of solution files are not prohibited, they
+          won't be included during a learning object's download.
```

Comment:
For consistency, we're trying to make sure that "Learning Object" is capitalized throughout the UI:

```suggestion
          won't be included during a Learning Object's download.
```